include/deal.II/lac/trilinos_tpetra_vector.h

```diff
@@ -125,15 +125,16 @@ namespace LinearAlgebra
        *
        * @ingroup TpetraWrappers
        */
-      template <typename Number>
+      template <typename Number, typename MemorySpace>
       class VectorReference
```

Comment:
Perhaps default the argument here as well? It probably doesn't matter given that people will unlikely create objects of this type by hand, but it won't hurt.